blog/discord-migration/discord-migration.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Fluree is moving to Discord
+tags: [fluree, Discord, community]
+description: Fluree is moving to discord; why, when, what-for
+---
+
+> Discord has better features for members and tooling for community management.  
+> We believe it will be a better platform for the fluree community.
```

Comment:
I think its important to start off with the points for why we are moving to discord, but maybe we should start off with a small blurb saying that the past year we have seen substantial growth (in terms of Fluree members/ users), due to this and due to some of our goals for this new year, we have unfortunately out grown our slack, then we can go into further reasons. 

I think building up excitement before we basically force our users to move might be a better motivator.

Reply:
thats a good point. ill try to incorporate that into the intro